api/lib/infrastructure/repositories/certification-center-repository.js

```diff
@@ -46,6 +46,21 @@ module.exports = {
       });
   },
 
+  getBySessionId(sessionId) {
```

Comment:
Je pense que tu devrais toujours faire du async/await, c'est plus joli. Je me doute que tu as voulu garder l'homogénéité dans le fichier, mais osef, go faire du code moderne ! On pourrait d'ailleurs imaginer un commit de BSR d'async/await des autres méthodes (le fichier n'est pas très gros, c'est rapide)